plugins/scaffolder-actions/scaffolder-utils/README.md

```diff
@@ -0,0 +1,194 @@
+# scaffolder-utils actions package
+
+Welcome to the roadie `utils` actions for the `scaffolder-backend`.
+
+This contains a collection of actions: 
+ - `roadiehq:utils:zip`
```

Comment:
I think maybe calling these:
 utils:zip might be enough. We already namespace the package, thats all the branding we need I would think.

Reply:
Yea, I was just thinking about adding namespacing here, because if you configure the scaffolder to be able to use your actions there you need to have a unique action ID for all of them. So it could actually happen that someone for some reason wants to have 2 zip actions setup on its backstage.

Reply:
In the absense of an agreed namespacing system for backstage scaffolder, we could stick our flag in the ground and claim the utils namespace?